### DISCUSSION TITLE
Why does the loss value become so high the second it has been trained exactly to the data

### DISCUSSION THREAD

**Discussion by superspaceshuttlee** [mrdbourke/pytorch-deep-learning#1117] [Q&A]
![image](https://github.com/user-attachments/assets/ceb4bcee-4a49-441c-97a9-6b327c0e58a9)

**Comment by LuluW8071** [comment]
Let's say you are learning how to ride a cycle, on your first try you definitely won't balance riding the cycle and you will have high chances of losing your balance. Slowly and gradually you will learn to balance and finally be able to ride it properly. And once you learn, you will be able to ride other cycles as well.

Similarly, any model you train at first is initialized with random parameter and it takes time to learn the features from data and generalize unseen data on that model

**Reply by superspaceshuttlee** [reply]
Thanks for the explanation
